content/automate/ManageIQ/Storage/Autosde/Services/ServiceVolumeRequestApproval.class/__methods__/approve_request.rb

```diff
@@ -0,0 +1,20 @@
+# Create a volume after service request
+
+$evm.log("info", 'creating a volume after order request.')
+ems = $evm.vmdb(:ext_management_system).where(:id=>$evm.root.attributes['dialog_ems']).first
```

Comment:
Minor, but `.where().first` should be `.find_by()` or in the case of `.find_by(:id => ...)` you can just use `.find($evm.root.attributes['dialog_ems'])`

Reply:
Fixed it